core/controllers/library_test.py

```diff
@@ -195,12 +195,12 @@ def test_thumbnail_icons_exist_for_each_category(self):
         # Test that an icon exists for each default category.
         for category in all_categories:
             utils.get_file_contents(os.path.join(
-                'static', 'images', 'subjects',
+                'static', utils.get_cache_slug(), 'images', 'subjects',
```

Comment:
Careful; this fails on Windows systems if the cache slug contains slashes.

That said -- what about defining the cache slug to be the thing after "prod/", and forcing it to have no slashes? Then the prefix in dev is 'dev' and the prefix in prod is 'prod/{{cache_slug}}'.